app/models/atmosphere/virtual_machine_template.rb

```diff
@@ -144,6 +144,12 @@ def perform_delete_in_cloud
       logger.warn("VMT with #{id_at_site} does not exist - continuing")
     end
 
+    # Returns the hourly cost for this template assuming a given VM flavor
+    def get_hourly_cost_for(flavor)
+      incarnation = flavor.flavor_os_families.find_by(os_family: appliance_type.os_family)
+      incarnation && incarnation.hourly_cost
+    end
```

Comment:
I see... It is used in `Atmosphere::OptimizationStrategy::Default#select` - so as @Nuanda correctly mentioned nil will lead to `NoMethodError: undefined method `<' for nil:NilClass` exception. 

Of course this is a situation when we will not have appropriate `os_family` in our DB. I propose to merge it and deploy it with guaranteeing correct `os_families` in our DBs and submit issue describing this problem which will be fixed ASAP. What do you think?